build/test.sh

```diff
@@ -51,8 +51,8 @@ fi
 echo
 
 check_dependencies() {
-    # Check if minio is already deployed
-    if helm status minio -n minio > /dev/null 2>&1 ; then
+    # Check if minio is already deployed. We suppress only StdOut for watching error if check will fail.
+    if helm status minio -n minio 1> /dev/null ; then
```

Comment:
Just for my understanding previous command was ignoring the stdout as well as stderr which would case actual error to be ignored. But the new command just ignores the stdout to make sure that we catch the stderr right?

Reply:
That's right.